src/components/Dashboard/ImportLecturesForm.js

```diff
@@ -29,7 +29,7 @@ const ImportLecturesForm = ({ course, onSaveComplete }) => {
     }),
     onSubmit: async values => {
       try {
-        await teacher.saveLecture({ ...values, course: course.id })
+        await teacher.importFromYoutube({ ...values, course: course.id })
```

Comment:
@abolkog the import from youtube was not linked to the correct API .. walla I am missing something?